.settings/.gitignore

```diff
@@ -0,0 +1,3 @@
+/org.eclipse.core.resources.prefs
```

Comment:
Could you please remove your personal .gitignore from repository?

Reply:
Done but any reason why we don't want it? If anyone is using eclipse then it might generate similar .gitignore which won't be added since it is already in git ignore. Assuming we let it add.

Reply:
Why we can't use common [.gitignore](https://github.com/folio-org/mod-finance-storage/blob/master/.gitignore) for this purpose? I think we can include Eclipse-specific files/folders there.

Reply:
So it already exists. Sounds good.